src/main/java/frc/robot/subsystems/conveyor/commands/Convey.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+import java.util.function.DoubleSupplier;
+
+public class Convey extends CommandBase {
```

Comment:
1. This name is confusing, what's the difference between "Cargo in" and "convey"?
2. These two commands are very similar. Perhaps you should combine them into a single command?